caesar_cipher.rb

```diff
@@ -0,0 +1,15 @@
+
+LETTERS_ARRAY = ("a".."z").to_a
+
+def caesar_cipher(string, shift_num)
```

Comment:
try running test with shift_num = 100 and see that happens :wink:

Reply:
done with `%`